test/e2e/framework/volume/fixtures.go

```diff
@@ -441,11 +441,11 @@ func runVolumeTesterPod(client clientset.Interface, config TestConfig, podSuffix
 	if slow {
 		err = e2epod.WaitForPodRunningInNamespaceSlow(client, clientPod.Name, clientPod.Namespace)
```

Comment:
this function indirectly uses https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/pod/wait.go where the pod specific timeouts are defined. Those timeouts should also eventually move to the timeouts.go. we can put a todo here?